moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeFeatureCollections.js

```diff
@@ -107,19 +76,11 @@ export const useAllComponentsFeatureCollection = (components) =>
         features: [],
       };
 
-    const allComponentsFeatures = [];
-
-    components.forEach((component) => {
-      const allComponentFeatures = getAllComponentFeatures(component);
-      allComponentsFeatures.push(allComponentFeatures);
-    });
+    const allComponentsFeatures = components.map((component) =>
+      getAllComponentFeatures(component)
+    );
 
-    // Make features valid GeoJSON by adding type and properties attributes
-    const geoJsonFeatures = allComponentsFeatures.flat().map((component) => ({
-      ...component,
-      type: "Feature",
-      properties: {},
-    }));
+    const geoJsonFeatures = allComponentsFeatures.flat();
```

Comment:
i had to shuffle some of the munging + flattening around to pull out `project_component_id` and get it attached to each `Feature`.